react/react1/week1/todo-list/src/components/ToDoList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+
+const initialTodos = [
+  { body: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+  { body: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+  { body: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+];
+
+function ToDoList() {
+  const [todoList, setTodos] = useState(initialTodos);
+
+  const getTodos = () => {
+    return todoList.map((todo) => {
+      return (
+        <div className="todos-container">
```

Comment:
I recommend you create another function component with the code contained in this div. :)

Reply:
Also, remember each child component has to have a key.

Reply:
Thanks for your time! I will change it in my week2 homework.